cmd/podman/containers/rename.go

```diff
@@ -14,8 +14,8 @@ var (
 		Short: "Rename an existing container",
 		Long:  renameDescription,
 		RunE:  rename,
-		Args:  cobra.ExactArgs(2), // TODO we can do better here - check that #1 is container ID, #2 is a name
-		//ValidArgsFunction: common.AutocompleteContainers,
+		Args:  cobra.ExactArgs(2),
+		ValidArgsFunction: common.AutocompletePortCommand,
```

Comment:
Is this correct?  PortCommand?

Reply:
@Luap99 recommended it - it completes the first argument as an existing container and does no completion on the second argument

Reply:
Ok we should get a different name for it then.

Reply:
Yes I will put this on my TODO list. I just need to find a descriptive name.

Reply:
Which can probably happen in a separate PR, because this needs to merge in the next 3-ish hours :smile: